gocd/web1devci.yaml

```diff
@@ -0,0 +1,73 @@
+pipelines:
+  pugin:
+    group: Dependencies
+    label_template: "${COUNT}"
+    environment_variables:
+      S3_BUCKET: s3://$(AWS_ACCOUNT).pugin-website
+      STATIC_ASSET_LOCATION_URL: https://s3-eu-west-1.amazonaws.com/$(AWS_ACCOUNT).static-assets
```

Comment:
Let's also add in the env `$(AWS_BUCKET)`, replicating the change I made to the Makefile. Will be good to test this change as will also need to be made eventually to the master branch web1live.yaml too.